apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/Models/Students.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Firestore.IntegrationTests.Models;
+
+[FirestoreData]
+public class Students : IEquatable<Students>
```

Comment:
Nit: this should be Student, not Students as each instance represents a single student.